website/docs/r/function_namespace.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "ibm"
+page_title: "IBM : function-namespace"
+sidebar_current: "docs-ibm-resource-function-namespace"
+description: |-
+  Manages IBM Cloud Functions namespace.
+---
+
+# ibm\_function_namespace
+
+Create, update, or delete [IBM Cloud Functions namespace](https://cloud.ibm.com/docs/openwhisk?topic=openwhisk-namespaces). You can create Identity and Access (IAM) managed namespaces to group entities, such as actions or triggers, together.
+
+## Example Usage
+
+### Create a namespace
+
+```hcl
+data "ibm_resource_group" "resource-group" {
+   name = var.resource_group
+}
+
+resource "ibm_function_namespace" "namespace" {
+   name                = var.namespace
+   resource_group_id   = data.ibm_resource_group.resource-group.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, string) The name of the namespace.
+* `description` - (Optional, string) Description of the namespace.
+* `resource_group_id` - (Required, string) The ID of the resource group.  You can retrieve the value from data source `ibm_resource_group`.
+* `resource_plan_id` - (Optional, string) The unique ID of the plan associated with the offering. This value is provided by and stored in the global catalog.
+* `location` - (Optional, string) Target location to create the namespace.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of the namespace.
+* `name` - The name of the namespace.
```

Comment:
No need to declare name, resource_group_id in attribute already part of argument